src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -42,21 +51,35 @@
     @GET
     @Path("/")
     @Produces(MediaType.APPLICATION_JSON)
-    public Collection<KabaneroInstance> getAllInstances() {
-        return KabaneroManager.getKabaneroManagerInstance().getAllKabaneroInstances();
+    public KabaneroList getAllInstances() throws IOException, ApiException, GeneralSecurityException {
+        KabaneroList kabaneros = KabaneroClient.getInstances();
+        List<Kabanero> kabaneroList = kabaneros.getItems();
+        if (kabaneroList.size() == 0){
+            return null;
```

Comment:
any need to return null here? We should just return an empty list instead (which _should_ translate to just an empty json array [])